File: src/main/java/fr/nozyx/bungeekotlin/BungeeKotlin.java
package fr.nozyx.bungeekotlin;

import net.md_5.bungee.api.plugin.Plugin;

public class BungeeKotlin extends Plugin {
    public void onEnable() {
        this.getLogger().info("BungeeKotlin started ! (Kotlin v2.0.0)");
    }

    public void onDisable() {
        this.getLogger().info("BungeeKotlin stopping!");
    }
}

File: src/main/kotlin/fr/nozyx/bungeekotlin/BungeeKotlin.kt
package fr.nozyx.bungeekotlin

import net.md_5.bungee.api.plugin.Plugin

class BungeeKotlin : Plugin() {
    override fun onEnable() {
        this.logger.info("BungeeKotlin started ! (Kotlin v2.0.0)")
    }

    override fun onDisable() {
        this.logger.info("BungeeKotlin stopping!")
    }
}